source/Octopurls.sln

```diff
@@ -44,6 +38,8 @@ Global
 		{F5CA833C-14F4-42CA-90A8-7D14DDA20C2A}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{F5CA833C-14F4-42CA-90A8-7D14DDA20C2A}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{F5CA833C-14F4-42CA-90A8-7D14DDA20C2A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{9A897B26-75CE-4B36-AA73-3B8AC9D931F7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Is this going to make IDE solution builds wonky?

Reply:
Nup. Tested that.